sufia-models/app/models/sufia/works/curation_concern/with_basic_metadata.rb

```diff
@@ -0,0 +1,35 @@
+# Basic metadata for all Works
+# Required fields:
+#   dc:title
+#   dc:rights
+#
+# Optional fields:
+#   dc:contributor
+#   dc:coverage
+#   dc:creator
+#   dc:date
+#   dc:description
+#   dc:format
+#   dc:identifier
+#   dc:language
+#   dc:publisher
+#   dc:relation
+#   dc:source
+#   dc:subject
+#   dc:type
+module Sufia::Works
+  module CurationConcern
+    module WithBasicMetadata
+      extend ActiveSupport::Concern
+
+      included do
+        include GenericWorkRdfProperties
```

Comment:
Unclear what you mean.  The files should be renamed, or sufia already has the concept?